content/reference/drops/context_drop.md

```diff
@@ -0,0 +1,14 @@
+# Context Drop
```

Comment:
It exists in one template, so might be worth documenting, just to prove it's no magic.